projects/hyperdash-angular/package.json

```diff
@@ -14,20 +14,20 @@
     "commit": "pushd ../.. && npm run commit"
   },
   "peerDependencies": {
-    "@angular/common": "^18.0.0",
-    "@angular/core": "^18.0.0",
-    "@angular/platform-browser": "^18.0.0",
-    "@angular-devkit/build-angular": "^18.0.0",
-    "core-js": "^3.37.1",
+    "@angular/common": "^19.0.5",
+    "@angular/core": "^19.0.5",
+    "@angular/platform-browser": "^19.0.5",
+    "@angular-devkit/build-angular": "^19.0.6",
```

Comment:
why is the version for build-angular different than other angular packages? Did we change it manually or through he ng update command?